src/main/java/core/basesyntax/IsoscelesTrapezoid.java

```diff
@@ -0,0 +1,29 @@
+package core.basesyntax;
+
+public class IsoscelesTrapezoid extends Figure {
+    private final double upperA; //Length of the upper base
+    private final double lowerB; // Length of the lower base
+    private final double height; // height
+
+    public IsoscelesTrapezoid(double upperA, double lowerB, double height) {
+        this.upperA = upperA;
+        this.lowerB = lowerB;
+        this.height = height;
+        setColor(new ColorSupplier().getRandomColor());
+    }
+
+    @Override
+    public void toDraw() {
```

Comment:
The method name 'toDraw' is not following the Java naming conventions. Consider renaming it to 'draw' to better reflect the action it performs and to comply with the checklist.